astropy/io/votable/connect.py

```diff
@@ -141,10 +143,12 @@ def write_table_votable(input, output, table_id=None, overwrite=False,
 
     # Tables with mixin columns are not supported
     if input.has_mixin_columns:
```

Comment:
I think `has_mixin_columns` is not really needed any more, so would suggest just removing this `if` statement.